README.md

```diff
@@ -1,19 +1,34 @@
-# About
+# About #
+**svneverever** helps you inspect the structure of a SVN repository and the changes made to it over time. Its most common use is in combination with [svn-all-fast-export](https://github.com/svn-all-fast-export/svn2git) (or "KDE's svn2git" if you wish).
 
-**svneverever** helps you inspect the structure of a SVN repository
-and the changes made to it over time.
-Its most common use probably is in combination with
-[svn-all-fast-export](https://github.com/svn-all-fast-export/svn2git)
-(or "KDE's svn2git" if you wish).
+# Installation #
 
+## Dependencies ##
+**svneverever** requires python 3.5 or higher and [pysvn](https://pysvn.sourceforge.io/). If you want to install **svneverever** with **pip** you need a few additional packages. For Debian/Ubuntu the following should work
 
-# Example
+```bash
+sudo apt install python3-setuptools python3-pip python3-svn
```

Comment:
python3-pip [depends on python3-setuptools](https://packages.debian.org/de/sid/python3-pip) and pip without setuptools would be sort of unusable. Sure the user needs pip but the interesting bits is that they need pysvn from their distro because PyPI doesn't have it. So in my world it's one additional package — pysvn — or two at most, if we consider it non-obvious how to obtain pip to use pip for people less familiar with Python. Let's reduce this to the part that deserves actual attention, at least get setuptools out, what do you think?